src/test/groovy/com/ableton/JenkinsMocksTest.groovy

```diff
@@ -1,18 +1,30 @@
 package com.ableton
 
 import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertNotNull
 import static org.junit.Assert.assertNull
 import static org.junit.Assert.assertTrue
 
 import com.lesfurets.jenkins.unit.BasePipelineTest
 import org.junit.After
+import org.junit.Before
 import org.junit.Test
 
 
 /**
  * Tests for the JenkinsMocks class.
  */
 class JenkinsMocksTest extends BasePipelineTest {
+  // The super class is not annotated with @Before since JenkinsPipelineUnit does not
```

Comment:
"The super class is not annotated with @Before" -> "The overridden method in not annotated with @Before", or "The method in the super class is not annotated with @Before".

Reply:
True, that's not very well-worded. Amended.